views/partials/dashboardResultsEApp.ejs

```diff
@@ -87,13 +76,13 @@
                         </a>
                     <% } %>
                 </td>
-                <td class="appService hide-col-sm hide-col-md">
+                <td class="hide-col-sm hide-col-md">
                     <%= users_applications[i].user_ref || 'Not available' %>
                 </td>
-                <td class="appService hide-col-sm hide-col-md">
```

Comment:
have you considered using the out of the box styles from the gov design system, e.g. "govuk-table__cell govuk-table__cell--numeric" for numeric cells which automatically right aligns the content?  Maybe easier than trying to code your own in informed.scss

https://design-system.service.gov.uk/components/table/

Reply:
Hey Ali, this is a good question. Unfortunately, this project is so old it uses an older version of the govuk sass classes which don't support the newer CSS classes for tables.

https://github.com/alphagov/govuk_elements/blob/master/assets/sass/elements/_tables.scss 

I guess we could update it but I don't want to mess up the existing tables. I think I might add a ticket to update it at some point in the future and find out if Kainos are okay to check my code, to make sure I haven't missed anything out. Would be a big job though 😅

Reply:
Actually, I take that back, it looks like this project has both the newer style code from `govuk_frontend_toolkit` and the old code from `govuk-elements-sass`, but since the older styles are at the bottom they will overwrite the new styles, so the proper way to fix it would be to remove all the old styles. Lol this project is a mess 🤦